src/main/java/com/gitrekt/resort/controller/FeedbackReportScreenController.java

```diff
@@ -0,0 +1,37 @@
+package com.gitrekt.resort.controller;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.stage.Stage;
+
+/**
+ * Handles the functionality of FeesbackReportScreen.fsml
+ */
+public class FeedbackReportScreenController implements Initializable {
+
+    @FXML
+    private Button backButton;
+    /**
+     * Initializes the controller class.
+     */
+    @Override
+    public void initialize(URL url, ResourceBundle rb) {
+        // TODO
+    }    
+    public void onBackButtonClicked() throws IOException {
+        Stage mainStage = (Stage) backButton.getScene().getWindow();
+        Parent staffHomeScreenRoot = FXMLLoader.load(
+            getClass().getResource("/fxml/ReportsHomeScreen.fxml")
+        );
+        Scene staffHomeScreen = new Scene(staffHomeScreenRoot);
+        mainStage.centerOnScreen();
```

Comment:
If you center the stage on the screen before you add the new scene to it, it will remain centered in the old position.